CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ All notable changes to this project will be documented in this file.
 ### Added
 
 - Field `arguments` added to the `amqp_0_9` input and output. (@calini)
+- New `elasticsearch_v8` output which supersedes the existing `elasticsearch` output that uses a deprecated Elasticsearch library. (@ooesili)
```

Comment:
Does it also supersede the `opensearch` output?

Reply:
No it does not, my understanding is elasticsearch and opensearch have diverged a bit so it's useful for us to support both